From 27f9cb89b6cc4f71dfe97d53537311ec7e9367c5 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Sun, 1 May 2022 23:57:11 +0200
Subject: [PATCH] Packaging: drop older releases from metainfo

Keep it simple. In following commits add also
autogenerated release date.

Signed-off-by: David Heidelberg <david@ixit.cz>
---
 data/cz.cvut.edu.comparch.qtrvsim.metainfo.xml.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/data/cz.cvut.edu.comparch.qtrvsim.metainfo.xml.in b/data/cz.cvut.edu.comparch.qtrvsim.metainfo.xml.in
index ae8afd0..1eae1cb 100644
--- a/data/cz.cvut.edu.comparch.qtrvsim.metainfo.xml.in
+++ b/data/cz.cvut.edu.comparch.qtrvsim.metainfo.xml.in
@@ -16,11 +16,6 @@
   </description>
 
   <releases>
-    <release version="0.9.2" date="2022-03-14">
-      <description>
-	<p>Initial Flatpak release.</p>
-      </description>
-    </release>
     <release version="@MAIN_PROJECT_VERSION@" date="2022-04-21">
       <description>
         <p>Current project release.</p>
-- 
2.35.1

